client/app/components/partials/SelectVariantAnnotationsDialog.vue

```diff
@@ -1,12 +1,16 @@
 <style lang="sass">
 @import ../../../assets/sass/variables
 
+.in-iframe .v-dialog--persistent
```

Comment:
This is dangerous. Without specifying a selector that limits the scope of this style to your dialog, you are overriding all dialogs. I'm going to reject this pull request and I will fix the styling so that it only impacts your dialog, not all dialogs.

Reply:
I created this problem with the style in FlaggedVariantsCard.vue. So I'm going to first fix the code there to make the style specific to each dialog. This is tricky because dialogs are not in a container that can easily be isolated with a css selector. @YangQi007, this is a bigger issue than just your dialog, so I will try to address this in a different manner.

Reply:
@tonydisera I am confused because the styling code above is from FlaggedVariantsCard. I don't why this code will affect my dialog component. If this styling code is only used in FlaggedVariantsCard component, it is supposed to be scoped. But for some reason, this styling code would effect my SelectVariantAnnotationsDialog. 

So do you mean make FlaggedVariantsCard scoped or make my component scoped?

Reply:
The problem is that the styling in FlaggedVariantCard applies globally to all persistent dialogs. This is my code and it is misleading. Because the style is placed in FlaggedVariantCard.vue does not automatically scope it. We would need to add the scope attribute to the style. But I hesitate to use local scoping because it is expensive and mostly unnecessary because most styles can be scoped by adding a selector of the parent container. This, however, doesn't work for dialogs. I'll try to find a better solution to this problem.